helm/kubernetes-node-exporter-chart/templates/daemonset.yaml

```diff
@@ -22,10 +22,20 @@ spec:
       tolerations:
       # Tolerate all taints for observability
       - operator: "Exists"
-      securityContext:
-        runAsUser: {{ .Values.userID }}
-        runAsGroup: {{ .Values.userGroup }}
       priorityClassName: giantswarm-critical
+      initContainers:
+      - name: {{ .Values.initContainer.name }}
+        image: {{ .Values.initContainer.image }}
+        command:
+        - /bin/sh
+        - -c
+        - "chown root:giantswarm /rootfs/var/lib/kubelet -R"
```

Comment:
Can't we change that in the host cluster in ignition?

Reply:
yeah good question, that would help

Reply:
I've tried that here https://github.com/giantswarm/k8scloudconfig/tree/kubelet_permissions 
But at the time folder is created and first pod is running wonership changes again. So kubelet may change it somewhere under the hood. Another option is to run kubelet as a different user, but that requires even more hacks

Reply:
I'll try some delay for service